src/main/java/org/broadinstitute/hellbender/utils/runtime/RuntimeUtils.java

```diff
@@ -31,4 +33,21 @@ public static File which(String executable) {
         }
         return null;
     }
+
+    /**
+     * Given a Class that is a CommandLineProgram, either GATK or Picard, return a display name suitable for
+     * presentation to the user that distinguishes GATK tools from Picard tools by including a " (Picard)" suffix;
+     * @param toolClass A CommandLineProgram class object may not be null
+     * @return tool display name
+     */
+    public static String toolDisplayName(final Class<?> toolClass) {
+        Utils.nonNull(toolClass, "A valid class is required to get a display name");
+        final String picardCommandLineClass = "picard.";
```

Comment:
I think it would be better to check that the class is a subclass of `picard.cmdline.CommandLineProgram`